### PR TITLE
Branch update about us kalaris

### PR DESCRIPTION
**Pull request by kalarisng**

### CODE REVIEW

File: docs/team/kalarisng.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    * PRs reviewed (with non-trivial review comments): [\#12](), [\#32](), [\#19](), [\#42]()
+    * Contributed to forum discussions (examples: [1](), [2](), [3](), [4]())
+    * Reported bugs and suggestions for other teams in the class (examples: [1](), [2](), [3]())
+    * Some parts of the history feature I added was adopted by several other class mates ([1](), [2]())
+
+* **Tools**:
+    * Integrated a third party library (Natty) to the project ([\#42]())
+    * Integrated a new Github plugin (CircleCI) to the team repo
+
+* _{you can add/remove categories in the list above}_
```
Comment (jundatan) [inline docs/team/kalarisng.md:50]: Please add one more empty line after this to solve EOF error.

**Review by jundatan — CHANGES_REQUESTED** [review]

File: docs/team/kalarisng.md (hunk 1 above)
Comment (kalarisng) [reply docs/team/kalarisng.md]: Okay, try the latest commit?

File: docs/team/kalarisng.md [hunk 2]
```diff
@@ -47,4 +47,5 @@ Given below are my contributions to the project.
     * Integrated a third party library (Natty) to the project ([\#42]())
     * Integrated a new Github plugin (CircleCI) to the team repo
 
-* _{you can add/remove categories in the list above}_
\ No newline at end of file
+* _{you can add/remove categories in the list above}_
+*
```
Comment (jundatan) [inline docs/team/kalarisng.md:51]: Can remove the asterisk (*) in the new line. 😆

**Review by jundatan — CHANGES_REQUESTED** [review]

File: docs/team/kalarisng.md (hunk 2 above)
Comment (kalarisng) [reply docs/team/kalarisng.md]: should work now!

**Review by jundatan — APPROVED** [review]

**Pull request merged**